app/lib/actions/katello/content_view_puppet_module/destroy.rb

```diff
@@ -29,9 +30,8 @@ def plan(repository)
             if content_view_puppet_modules.any?
               puppet_repoids = ::Katello::Repository.puppet_type.in_environment(repository.organization.library).
                   pluck(:pulp_id).reject { |repoid| repoid == repository.pulp_id }
-              found_puppet_module = ::Katello::PuppetModule.latest_modules_search([{:name => puppet_module.name,
-                                                                                    :author => puppet_module.author}],
-                                                                                  puppet_repoids).first
+              found_puppet_module = ::Katello::PuppetModule
+                .latest_module(puppet_module.name, puppet_module.author, puppet_repoids)
```

Comment:
The way you've broken apart these lines is a bit inconsistent as line 13 seems to have a `.` but line 33 does not. 

Also, I personally like 80 char line limit but I think we've been using 100 chars as a rule of thumb on katello.